src/application.cpp

```diff
@@ -453,8 +453,8 @@ Document *Application::createDocument(const QString &label)
 {
     auto doc = new Document(this);
     if (label.isEmpty()) {
-        static std::atomic<unsigned> docSequenceId = 0;
-        const unsigned docId = std::atomic_fetch_add(&docSequenceId, 1);
+        static std::atomic<unsigned> docSequenceId = {0};
+        const unsigned docId = std::atomic_fetch_add(&docSequenceId, (unsigned)1);
```

Comment:
Please do it like this actually :
const unsigned docId = docSequenceId.fetch_add(1);

This has the advantage to circumvent the explicit type cast.
(Tested with GCC-5.1.0)